lib/collections/addon/components/collections-submission/template.hbs

```diff
@@ -74,69 +78,85 @@
-                    @discard={{action 'noop'}}
-                    @continue={{action 'nextSection'}}
+                    @discard={{action this.noop}}
+                    @continue={{action this.nextSection}}
                 />
-            {{/section.active}}
-            {{#section.complete}}
-                <p>{{contributor-list node=this.collectionItem shouldTruncate=false}}</p>
-            {{/section.complete}}
-        {{/sections.section}}
+            </section.active>
+            <section.complete>
+                <p>
+                    <ContributorList
+                        @node={{this.collectionItem}}
+                        @shouldTruncate={{false}}
+                    />
+                </p>
+            </section.complete>
+        </sections.section>
 
-        {{#sections.section
-            section=this.sections.collectionMetadata
-            tooltip=(t (concat this.i18nKeyPrefix 'closed_tooltip'))
-            title=(t (concat this.i18nKeyPrefix 'collection_metadata_title'))
+        <sections.section
+            @section={{this.sections.collectionMetadata}}
+            @tooltip={{t (concat this.i18nKeyPrefix 'closed_tooltip')}}
+            @title={{t (concat this.i18nKeyPrefix 'collection_metadata_title')}}
             as |section|
-        }}
-            {{#section.active}}
+        >
+            <section.active>
                 <CollectionMetadata
                     data-test-collection-metadata
                     @collection={{this.collection}}
                     @collectedMetadatum={{this.collectedMetadatum}}
-                    @continue={{action 'nextSection'}}
+                    @continue={{action this.nextSection}}
                 />
-            {{/section.active}}
-            {{#section.complete}}
+            </section.active>
+            <section.complete>
                 {{#each this.choiceFields as |field|}}
                     <p>
                         <em>{{t field.label}}</em>
                         <span>{{field.value}}</span>
                     </p>
                 {{/each}}
-            {{/section.complete}}
-        {{/sections.section}}
+            </section.complete>
+        </sections.section>
 
         <section class='row' local-class='last-buttons'>
             <div class='col-xs-12 text-right'>
-                <button class='btn btn-default' {{action 'cancel'}}>
+                <button class='btn btn-default' {{action this.cancel}}>
                     {{t (concat this.i18nKeyPrefix 'cancel')}}
                 </button>
                 {{#if this.edit}}
-                    <button data-test-collection-update data-analytics-name='Update' class='btn btn-success' {{action (perform this.save)}}>
+                    <button
+                        data-test-collection-update
+                        data-analytics-name='Update'
+                        class='btn btn-success'
+                        {{action (perform this.save)}}
```

Comment:
Remove action here and just have perform

Reply:
SAA